packages/react/src/Header/Header.tsx

```diff
@@ -11,54 +11,49 @@ import { Logo } from '../Logo'
 import type { LogoBrand } from '../Logo'
 
 export type HeaderProps = {
-  /** The name of the application. */
+  /** A site-wide title for the website or application. */
   appName?: string
-  /** The list of menu links. Use a Page Menu here. */
-  links?: ReactNode
-  /** The name of the brand for which to display the logo. */
+  /** The HTML element to use. */
+  as?: 'header' | 'div'
+  /** The brand for which to display the logo. */
   logoBrand?: LogoBrand
   /** The url for the link on the logo. */
   logoLink?: string
   /** The accessible text for the link on the logo. */
   logoLinkTitle?: string
-  /** A button to toggle the visibility of a Mega Menu. */
+  /** A secondary menu of links and buttons. Use a Page Menu. Can include buttons to show the Mega Menu or a Search Field. */
   menu?: ReactNode
-} & HTMLAttributes<HTMLElement>
+} & HTMLAttributes<HTMLDivElement>
 
 export const Header = forwardRef(
   (
     {
       appName,
+      as: Tag = 'div',
```

Comment:
This is not always a `<header>`? There are also a11y factors to consider, like the definition of the banner role. So you would need a `<div role="banner">` to replace it correctly.

Reply:
No, this approach separates the `Header` and `Mega Menu` components entirely and leaves it to the app to let the menu button in the header toggle the menu's visibility.

And because the menu should be part of the HTML `header`, the `Header` component can no longer render that element – unless there is no Mega Menu, in which case people can do `<Header as="header">`.

The example Home Page already uses this approach, and I’ll document it further.

Reply:
This could solve it for now, Grid also defaults to div so not needed.
```suggestion
      as: Tag = 'header',
```

Reply:
The typical setup will combine a `Header` with a `MegaMenu`. Making `header` the default for the former sets people up for mistakes.

The Grid wrapping the Header and the Mega Menu will need to render the `header`.

Only if a site doesn’t use a Mega Menu (or Search button in the Page Menu, or anything else) must the Header render a `header`. Having the least common option as the default will set people up for mistakes.

Reply:
> And because the menu should be part of the HTML header

It turns out this is not required – the Header can render a `<header>` and the Mega Menu a `<nav>` below it.

Reply:
Perhaps it's still in my old PR's, I was told both the menu button, page-menu and mega-menu should reside in the same nav element, like gov.uk does it. This presented my with a lot of challenges so the PR was dropped. I'm curious what changed? @alimpens could you elaborate?

Reply:
Found the comment with the link: https://github.com/Amsterdam/design-system/pull/992#discussion_r1445836227